lib/cmake/scipp-config.cmake.in

```diff
@@ -4,12 +4,25 @@
 # ~~~
 @PACKAGE_INIT@
 
-include("${CMAKE_CURRENT_LIST_DIR}/scipp-targets.cmake")
+# Call find_package(scipp ... COMPONENTS conan-config) to use conan for
+# install of dependencies. Then call a second time without COMPONENTS to
+# get the actual package, with dependencies. If system dependencies should
+# be used instead the call with "conan-config" can be omitted.
+if(scipp_FIND_COMPONENTS)
+  foreach(_comp ${scipp_FIND_COMPONENTS})
+    if (NOT _comp STREQUAL "conan-config")
+      message(FATAL_ERROR "bad")
+    endif()
+  endforeach()
+  include("${CMAKE_CURRENT_LIST_DIR}/scipp-conan.cmake")
+else()
+  include("${CMAKE_CURRENT_LIST_DIR}/scipp-targets.cmake")
 
-check_required_components(scipp-targets)
+  check_required_components(scipp-targets)
 
-include(CMakeFindDependencyMacro)
-find_dependency(Boost 1.69)
-find_dependency(Eigen3)
-find_dependency(LLNL-Units)
-find_dependency(TBB)
+  include(CMakeFindDependencyMacro)
+  find_dependency(Boost 1.69)
```

Comment:
This version does not match the one in `scipp-conan.cmake`. Does it have to be specified here?

Reply:
This here defines the minimum required version. `scipp-conan.cmake` defines the one you get when you use conan to install boost. If someone wants to use scipp without conan they still need 1.69 or later, so it has to be specified here.